### PR TITLE
Libre2 Documentation

### PR DESCRIPTION
**Pull request by jk1993**

### CODE REVIEW

File: LIBRE2.md [hunk 1]
```diff
@@ -0,0 +1,193 @@
...
+
+If you do not have git installed, you will need to install that too:
+
+    sudo apt-get install -y git
+
+To run the program manually do: 
+
+1. git clone https://github.com/tzachi-dar/LibreAllHouse.git
+2. cd LibreAllHouse/
+3. Edit the config.cfg file to unter mongodb parameters (optional). (for example db_uri = mongodb+srv://userj:passi@cluster-b6k1n0tj.bg18m.mongodb.net/heroku_b6k1n0tj)
```
Comment (tzachi-dar) [inline LIBRE2.md]: Due to the need for a shared key, libre2 does not work with mongodb. So, remove section 3.

File: LIBRE2.md [hunk 2]
```diff
@@ -0,0 +1,193 @@
...
+    sudo pip3 install bluepy
+    sudo pip3 install pymongo
+    sudo pip3 install dnspython
+    sudo apt-get install -y vim
+
+## Step 4 - Run the program:
+
+If you do not have git installed, you will need to install that too:
+
+    sudo apt-get install -y git
```
Comment (tzachi-dar) [inline LIBRE2.md:51]: Move this part to step 3

File: LIBRE2.md [hunk 3]
```diff
@@ -0,0 +1,193 @@
...
+Ctrl+C to cancel monitoring log 
+ 
+## Step 5 - Configure xDrip to connect to PI
+
+1.  On Hardware Data Source select ‘Libre wifi’ or ‘Libre Bluetooth + wifi’
+    
+2.  On the list of receivers, enter the list of raspberry pis and/or the mongodb addresses. The raspberry pi software runs on port 50005 so you need to include this in the address For example:
+
+`192.168.1.2:50005,mongodb://user:[pass@ds11551.mlab.com](mailto:pass@ds11551.mlab.com):11551/nightscout3/libre`
+
```
Comment (tzachi-dar) [inline LIBRE2.md]: Remove mongodb from this step as well.

File: LIBRE2.md [hunk 4]
```diff
@@ -0,0 +1,193 @@
...
+	`sudo pico config.cfg`
+
+2. Change the **api_secret** property to be anything you like (remember this secret)
+3. Save the config file
+4. In xdrip, go to `Settings -> Inter-app settings`.
+5. Enable **xDrip Web Service**
+6. Check the box for **Open Web Service**
+7. Enter an **xDrip Web Service Secret**. This should match the secret configured in the **config.cfg** file on your Pi
+
+## Step 7 - Configure the pi to also upload data to a mongodb (optional).
```
Comment (tzachi-dar) [inline LIBRE2.md]: Remove step 7

File: LIBRE2.md [hunk 5]
```diff
@@ -0,0 +1,193 @@
...
+
+[MongoData]
+
+db_uri = mongodb+srv://USER:PASSWORD@URLPREFIX.mongodb.net
+
+db_name = DBNAME
+
+collection_name = COLLECTION
+
+## Step 8 - Configure a phone as receiver (optional)
```
Comment (tzachi-dar) [inline LIBRE2.md]: Remove step 8, it does not work for libre2

**Review by tzachi-dar — CHANGES_REQUESTED** [review]

**Comment by jk1993** [comment]
I have made the requested changes. Let me know if I've missed anything. Thanks

**Comment by tzachi-dar** [comment]
Thanks for your help!

**Pull request merged**